README.md

```diff
@@ -5,17 +5,9 @@ This is a simple container wrapping libvirtd.
 ## Purpose
 
 Instead of delivering libvirtd in a package, libvirtd is
-now delivered in a container.
-When running the container it will _look_ like libvirtd is
-running on the host's namespace. But in fact it's running
-in a container.
-
-This is similar to Atomic's 'system containers'.
-
-You can use virsh on the host to access it.
-
-To access libvirtd via TCP you still need to export
-the correct port.
+now delivered in a container. This container is used as
+the base container image for `virt-launcher` to run
+as the entrypoint, it is not intended to be run standalone.
 
 ## Versions to use
```

Comment:
The entire "Versions to use" section is now unnecessary as well - we literally just told the user they're **not** supposed to run the container outside of KubeVirt :)

Reply:
Dropped now